setup.ts

```diff
@@ -116,6 +117,24 @@ async function accessAndRefreshTokens(
   }
 }
 
+//set the image size upload environment variable
+/**
+ * The function `setImageUploadSize` sets the image upload size environment variable and changes the .env file
+ * @returns The function `checkExistingRedis` returns a void Promise.
+ */
+async function setImageUploadSize(size: number): Promise<void> {
```

Comment:
How are negative and zero value numbers going to be handled?

Reply:
```
const { imageSizeLimit } = await inquirer.prompt([
    {
      type: "input",
      name: "imageSizeLimit",
      message: `Enter the maximum size limit of Images uploaded (in MB) max: ${
        MAXIMUM_IMAGE_SIZE_LIMIT_KB / 1000
      }`,
      default: 3,
      validate: (input: number) =>
        validateImageFileSize(input) ||
        `Enter a valid number between 0 and ${
          MAXIMUM_IMAGE_SIZE_LIMIT_KB / 1000
        }`,
    },
  ]);
```
while taking the input it firstly validates using the "validateImageFileSize" function
```
function validateImageFileSize(size: number): boolean {
  return size > 0;
}
```